src/views/Project/index.jsx

```diff
@@ -116,14 +120,21 @@ export default class Project extends Component {
           <Typography variant="display2" align="center">
             {project.name}
           </Typography>
+          {project.introduction && (
+            <ExpansionPanel>
+              <ExpansionPanelSummary expandIcon={<ChevronDownIcon />}>
+                <Typography>Project Introduction</Typography>
```

Comment:
I think we want a `variant="display1"` to make sure `Project Introduction` is bigger in size than markdown.